### DISCUSSION TITLE
Bounding boxes

### DISCUSSION THREAD

**Discussion by erkin98** [jsvine/pdfplumber#375] [Q&A] [answered]
How to find bounding boxes coordinates(x0, top, x1, bottom)?

**Comment by samkit-jain** [comment, marked as answer]
Hi @erkin98 Could you please elaborate?

**Reply by erkin98** [reply]
Returns a version of the page cropped to the bounding box, which should be expressed as 4-tuple with the values (x0, top, x1, bottom).
İ want to learn how to find this coordinates?

**Reply by samkit-jain** [reply]
>Returns a version of the page cropped to the bounding box, which should be expressed as 4-tuple with the values (x0, top, x1, bottom).

This text is from the description of the `page.crop(...)` method which means that you want to know how to construct this tuple. First of, these 4 values are

Property | Description
-- | --
x0 | Distance of left side of rectangle from left side of page.
top | Distance of top of rectangle from top of page.
x1 | Distance of right side of rectangle from left side of page.
bottom | Distance of bottom of the rectangle from top of page.

If you want to crop the page to a particular object, say, a table, you can get the bounding box of the table using the `bbox` property. Example, `page.find_tables()[index].bbox`. If you want to crop in terms of percentages, for example, to crop the page and remove the 10% margin from all sides, use `(0.1 * float(p.width), 0.1 * float(p.height), 0.9 * float(p.width), 0.9 * float(p.height))`.

**Reply by erkin98** [reply]
thanks :)

**Reply by julian-r** [reply]
What is the unit of the measurment? pixel? if yes what is the DPI?

**Reply by jsvine** [reply]
The units throughout `pdfplumber` and `pdfminer.six` are "[points](https://guides.gdpicture.com/content/About%20a%20PDF%20format.html)," which for PDFs default to 1/72 of an inch.

**Comment by ancy18** [comment]
I'm using this to extract the table from pdf. I've to find the bounding box of each cell in the table and push it into csv. How to do that?

**Reply by samkit-jain** [reply]
You can use the `.find_tables(...)` method which returns a list of `Table` objects. The `Table` object provides access to the `.cells` property which can give you access to the bounding box of a cell.
